lib/FullScreenView/FullScreenView.js

```diff
@@ -0,0 +1,78 @@
+import React, {
+  useRef,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Pane,
+  PaneMenu,
+  IconButton,
+  Button,
+} from '@folio/stripes/components';
+
+import css from './FullScreenView.css';
+
+export const FullScreenView = ({
+  id = 'full-screen-view',
+  children,
+  paneTitle,
+  onCancel,
+}) => {
+  const headerCloseButtonRef = useRef();
+
+  useEffect(() => headerCloseButtonRef.current.focus(), []);
+
+  const firstMenu = (
+    <PaneMenu>
+      <FormattedMessage id="stripes-components.cancel">
+        {ariaLabel => (
+          <IconButton
+            data-test-header-close-button
+            ariaLabel={ariaLabel}
+            icon="times"
+            ref={headerCloseButtonRef}
+            onClick={onCancel}
+          />
+        )}
+      </FormattedMessage>
+    </PaneMenu>
+  );
+
+  return (
+    <div
+      id={id}
+      data-test-full-screen-view
+    >
+      <Pane
+        defaultWidth="100%"
+        paneTitle={paneTitle}
+        firstMenu={firstMenu}
+        actionMenu={({ onToggle }) => (
+          <Button
```

Comment:
It seems, I didn't understand the idea. @Vladyslav-Velytskyi  please, could you show (the screenshot) how this button looks like?

Reply:
![image](https://user-images.githubusercontent.com/55701515/84484151-04e53b80-aca3-11ea-9edc-64cd52681b76.png)
This is Action button, actually implementing of this button is out of scope of this story, it must be implementing in [UIDEXP-75](https://issues.folio.org/browse/UIDEXP-75)

Reply:
It should work like a temp solution for Q2. Anyway, PR looks good for me